tidy3d/web/simulation_task.py

```diff
@@ -537,3 +537,9 @@ def get_log(
             verbose=verbose,
             progress_callback=progress_callback,
         )
+
+    def abort(self):
+        """Abort current task from server."""
```

Comment:
Same as for fdtd, "Abort the solver execution if task is currently running."

Reply:
Updated CHANGELOG